lineman/app/css/1_utilities.scss

```diff
@@ -109,11 +109,15 @@ label.label-with-details{
   padding: 10px;
 }
 
-.lmo-h1{
+@mixin lmoH1{
```

Comment:
I want to flag that I'm not a fan of this BEM mixin strategy, because it makes us bury through another layer of abstraction before getting to the actual styles.

ie,

.user-page__name . o (WTH is that?) -> @mixin lmoH1. o (WTH is that?) -> actual styles

instead of

.lmo-h1 . o (WTH is that?) -> actual styles